src/encoder.rs

```diff
@@ -590,10 +980,13 @@ mod tests {
                         w: &mut out,
                     };
 
-                    let mut encoder = Encoder::new(&mut wrapper, info.width, info.height)
+                    let _ = Encoder::new(&mut wrapper, info.width, info.height)
                         .write_header()
+                        .unwrap()
+                        .next_image()
+                        .unwrap()
+                        .write_image_data(&buf)
```

Comment:
This makes it pretty clear that the changes are an ergonomic loss. A method call without parameters, between two unwraps/errors is _not_ what you want the common, simple usage to look like.